### PR TITLE
Update Valheim Plus to use game specific image and support crossplay

### PR DESCRIPTION
**Pull request by Shigbeard**
# Description

Updates Valheim Plus egg to use the game specific image as already used in Valheim Vanilla, adds support for crossplay, and uses the PDTL_V2 format.

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### CODE REVIEW

**Comment by Shigbeard** [comment]
Note: this will still not function due to an issue with Valheim Plus not being updated. This problem is out of scope due to the fact that they haven't pushed an update for the latest version of Valheim to their repo. They have a pending pull request from one of their developers that resolves the issue but due to work commitments, the repo maintainer cannot approve the pull request for a bit.

This is documented in their Discord server, and once they've resolved this issue, this egg will work as intended. In the interim, users are expected to replace the DLL file obtained automatically for ValheimPlus with an alpha release version uploaded to their discord.

**Comment by Shigbeard** [comment]
https://github.com/valheimPlus/ValheimPlus/pull/710

The maintainers of ValheimPlus have merged the pull request that makes it compatible with the latest release of Valheim, which means by the time you're reading this, this egg will be functioning completely normally, without the need for replacing DLLs once the server is installed.

File: game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json
Comment (parkervcp) [inline game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json:27]: Why was this changed? It looks like you just copied it from another egg.

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json
Comment (danny6167) [inline game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json:18]: You've removed the part that passes the $PUBLIC variable

File: game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json
Comment (gOOvER) [reply game_eggs/steamcmd_servers/valheim/valheim_plus/egg-valheim-plus-mod.json:27]: looks like, he pushed my egg, what i send him of Discord pn

**Comment by QuintenQVD0** [comment]
I prefer #1914 as it is a all in one fix

**Comment by QuintenQVD0** [comment]
This is closed because #1914 Got merged and it was a all in one fix that did not need changes but still thanks for your time!

**Pull request closed without merging**
